pkg/analysis/nophase/testdata/src/a/a.go

```diff
@@ -0,0 +1,13 @@
+package a
+
+type NoPhaseTestStruct struct {
+	// +optional
+	Phase *string `json:"phase,omitempty"` // want "field Phase: phase fields are deprecated and conditions should be preferred, avoid phase like enum fields"
```

Comment:
What about a field that didn't include phase in the Go name, but did include it in the serialised name?

Reply:
Hmm. I hadn't thought about that - we'd need to parse the jsontags then?

Reply:
we now parse jsontags